Gemfile

```diff
@@ -116,3 +116,5 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem "phonelib"
+
+gem "concurrent-ruby", "~> 1.1"
```

Comment:
Would you want to add a short comment on why this dependency was included, i.e. what feature needed it?